templates/cloudflare-workers/vite.config.ts

```diff
@@ -0,0 +1,17 @@
+import react from '@vitejs/plugin-react-swc'
+import { defineConfig } from 'vite'
+
+// https://vitejs.dev/config/
+export default defineConfig(({ mode }) => {
+	if (mode === 'production' && !process.env.TLDRAW_WORKER_URL) {
+		throw new Error('TLDRAW_WORKER_URL must be set in production')
+	}
+
+	return {
+		plugins: [react()],
+		define: {
+			'process.env.TLDRAW_WORKER_URL':
+				process.env.TLDRAW_WORKER_URL ?? '`http://${location.hostname}:5172`',
```

Comment:
```suggestion
				process.env.TLDRAW_WORKER_URL ?? '`http://${location.host}`',
```
i think we can do this and avoid hardcoding the port in too many places

Reply:
the port here is different: it's 5173 for the client, and 5172 for the server

Reply:
silly me